### DISCUSSION TITLE
would it be possible to get support for Pypy  / Hpy?

### DISCUSSION THREAD

**Discussion by stonebig** [duckdb/duckdb#4461] [General]
I don't see wheels for PyPy on pypi, for windows, so I suppose it's maybe not yet compatible ?

**Comment by Mause** [comment]
PyPy does look possible with some minor fixes, but HPy support wouldn't be possible without replacing pybind11

Happy to merge a PR that fixes the issues with PyPy 😁

**Comment by fomcl** [comment]
@Mause could you elaborate on what is not yet working under Pypy? Pypy is supposed to be a drop-in replacement for CPython, after all.

**Comment by Mause** [comment]
@fomcl you can look at the build failure here: https://github.com/Mause/duckdb/runs/7709633572?check_suite_focus=true

**Comment by Mause** [comment]
Looks like we were once compatible though: https://github.com/duckdb/duckdb/pull/720

**Comment by fomcl** [comment]
Well, I do see a  `WARNING: The wheel package is not available.` in the log you provided.
